api/SubtleCrypto.json

```diff
@@ -728,7 +765,8 @@
               "version_added": "37"
             },
             "edge": {
-              "version_added": "12"
+              "version_added": "12",
+              "partial_implementation": true
```

Comment:
This marks the method as partial but doesn't list the primitives that are not supported in Edge. The table in the spec lists some primitives that are not supported, is there a reason they're omitted here?

Similarly  `exportKey()` isn't marked as partial, but seems to also have some primitives that are not supported in Edge.

Reply:
I wasn't sure what to do with this, since it's unclear to me whether `importKey`/`exportKey` will necessarily fail, or simply give you an object you can't do anything with. But I suppose that if an interface is marked `partial` it really should have a note explaining why.

Reply:
Yes, it's a good question. This: https://github.com/diafygi/webcrypto-examples seems like a really helpful resource.

I just tested in BrowserStack, and this: https://github.com/diafygi/webcrypto-examples#aes-ctr---importkey throws an error in Edge (but not in Firefox), while this: https://github.com/diafygi/webcrypto-examples#aes-cbc---importkey does not throw in Edge.